src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -396,4 +397,55 @@ BOOST_AUTO_TEST_CASE(DiceLossTest)
   BOOST_REQUIRE_EQUAL(output.n_cols, input2.n_cols);
 }
 
+/*
+ * Simple test for the Triplet Margin Loss function.
+ */
+BOOST_AUTO_TEST_CASE(TripletMarginLossTest)
+{
+  arma::mat anchor, positive, negative;
+  arma::mat input, target, output;
+  TripletMarginLoss<> module;
+
+  // Test the Forward function on a user generated input and compare it against
+  // the manually calculated result.
+  anchor = arma::mat("2 3 5");
+  positive = arma::mat("10 12 13");
+  negative = arma::mat("4 5 7");
+
+  input = {
```

Comment:
I think we can put this in a single line?

Reply:
I did it so it is easy to see it's a matrix of dimensions 3x2. I can change that if you say.